proto/bragi/common.proto

```diff
@@ -49,25 +49,40 @@ message ControlErrorAnnouncement {
   optional string message = 2;
 }
 
+message MatchTimeline {
+  repeated Match matches = 1;
+}
+
 message Match {
   string match_urn = 1;
-  string tournament_urn = 2;
+  string tournament_urn = 2 [deprecated=true]; // Deprecated. Use tournament.
   Sport sport = 3;
-  string home_team_urn = 4;
-  string away_team_urn = 5;
+  string home_team_urn = 4 [deprecated=true]; // Deprecated. Use competitors.
+  string away_team_urn = 5 [deprecated=true]; // Deprecated. Use competitors.
   uint32 home_score = 6;
   uint32 away_score = 7;
   MatchStatusType match_status = 8;
   google.protobuf.Timestamp plannet_start = 9;
+  Tournament tournament  = 10;
+  repeated Competitor competitors = 11;
 }
 
-message MatchTimeline {
-  repeated Match matches = 1;
+message Tournament {
+  string id = 1;
+  string name = 2;
 }
 
-message Team {
-  string team_urn = 1;
-  string team_name = 2;
+message Competitor {
```

Comment:
Suggestion 2, something like: 

```
message Competitor {
   oneof type {
      Teams teams = 1,
      Competitors competitors = 2,
   }
}
message Teams {
   string home _urn= 1;
   string away_urn = 2;
}
message Competitors {
string urn = 1;
}
```

Reply:
But for this case, we can have more than two teams especially in the racing markets